makefile

```diff
@@ -145,7 +145,7 @@ apidocs:
 	cp doc/doxygen.css api_docs/html/
 	@echo "\n=> Open up 'api_docs/html/index.html' in a browser to view a local copy of the documentation"
 
-STYLIZE_DIFFBASE ?= master
+STYLIZE_DIFFBASE ?= staging
```

Comment:
This makes sense for speeding up checkstyle and pretty, but how did this make CI fail?